internal/backend/sessions/sessionworkflows/workflows.go

```diff
@@ -87,6 +87,7 @@ func (ws *workflows) StartWorkers(ctx context.Context) error {
 }
 
 func (ws *workflows) StartWorkflow(ctx context.Context, session sdktypes.Session) error {
+	fmt.Println("  !!! session: ", session.ID(), session.CreatedAt())
```

Comment:
no fmt please, if you need log it with debug
if it was for development only, please remove

Reply:
already removed

Reply:
already removed